config/environments/production.rb

```diff
@@ -131,4 +131,20 @@
 
   # see https://discuss.rubyonrails.org/t/cve-2022-32224-possible-rce-escalation-bug-with-serialized-columns-in-active-record/81017
   config.active_record.yaml_column_permitted_classes = [Time]
+
+  # From https://github.com/paper-trail-gem/paper_trail/wiki/Setting-whodunnit-in-the-rails-console
+  console do
+    PaperTrail.request.whodunnit = lambda {
+      @paper_trail_whodunnit ||= begin
+        email = nil
+        until email.present?
+          # rubocop:disable Rails/Output
+          puts "Enter your email address for PaperTrail"
```

Comment:
Wonder if there's anyway to automatically get this based on the user used to initiate the ssh session

Reply:
Good idea. I'll have a look.